schemas/applications/schema/hardware.schema.json

```diff
@@ -26,6 +26,24 @@
             250
           ]
         },
+        "tf_prefix": {
+            "description": "The TF prefix to use for the URDF",
+            "type": "string",
+            "default": "",
+            "examples": [
+              "my_robot_",
+              "left_robot_",
+              "robot_1_"
+            ]
+        },
+        "reference_frame": {
+            "description": "The reference frame to use for the URDF",
+            "type": "string",
+            "default": "",
```

Comment:
Here I would omit the default tag; not supplying a reference frame is the same as using the root link of the URDF as the reference frame, which is different than saying `reference_frame == ""`
```suggestion
```